torchaudio/functional.py

```diff
@@ -524,34 +524,38 @@ def lfilter(waveform, a_coeffs, b_coeffs):
 
     Returns:
         output_waveform (torch.Tensor): Dimension of `(n_channel, n_frames)`.  Output will be clipped to -1 to 1.
+                                        Will be on the same device as the inputs.
```

Comment:
This is usually assumed. We probably don't have to say it here, but it's good to keep in mind :)